c7n/resources/ebs.py

```diff
@@ -403,6 +403,32 @@ class resource_type(object):
     action_registry = actions
 
 
+@EBS.action_registry.register('detach')
+class VolumeDetach(BaseAction):
+
+    """
+    Detach an EBS volume from an Instance.
+
+     :example:
+
+         .. code-block:: yaml
+
+         policies:
+           - name: instance-ebs-volumes
+             resource: ebs
+             filters: [{'VolumeId' :  'volumeid'}]
+
+
+    """
+
```

Comment:
all filters/actions need a permission attribute denoting what iam permission they need, and also a schema attributing defining their configuration. Full ci has a few tests that ensure those conditions are met.

Also we probably want a force boolean option here.